docs/ADVANCED.md

```diff
@@ -104,6 +104,61 @@ func loadMigrationsFromDir(dir string) []string {
 **Use cases**: Rollback support, conditional migrations, multi-schema setups,
 external migration sources.
 
+## Usage in multiple packages
+
+As described in [the `go test` documentation][go-test-documentation],
+running `go test ./...` causes the execution of one test binary per package.
+By default, the test binaries are run in parallel, which can lead to issues when
+multiple test binaries attempt to create a template database on the same
+PostgreSQL server.
```

Comment:
It would be nice to specify which issues arise because even though there are multiple test binaries, they are run within one process and n threads. Consequently, the `sync.Once` mechanism is good enough to work. Is my understanding correct?

I think it makes sense to see how the usage of `liquibase` (please let me know if the spelling is wrong) changes this behaviour and how you have debugged there are several Postgres containers in your testing environment.